helloextend-protection/includes/class-helloextend-global.php

```
@@ -287,15 +310,12 @@ public function update_price($cart_object)
         $cart_items = $cart_object->cart_contents;
 
         if (!empty($cart_items)) {
-            foreach ($cart_items as $value) {
-                if (
-                    isset($value['extendData'], $value['extendData']['price'], $value['data']) &&
-                    is_numeric($value['extendData']['price']) &&
-                    is_object($value['data'])
-                ) {
-                    $value['data']->set_price(round(((float) $value['extendData']['price']) / 100, 2));
+
+            foreach ($cart_items as $key => $value) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$key'. (undefined)

(UnusedLocalVariable)